packages/rollup-plugin-html/test/src/input/extract/extractAssetsByTagAndAttributes.test.ts

```diff
@@ -0,0 +1,154 @@
+import { expect } from 'chai';
```

Comment:
This test is a copy from ./extractAssets.test.ts, modified to include the new tagAndAttributes feature.